wavenet.py

```diff
@@ -88,20 +93,34 @@ def _create_dilation_layer(self,
             [1, dilation_channels, in_channels], stddev=0.2, name="dense"))
         transformed = tf.nn.conv1d(out, weights_dense, stride=1,
                                    padding="SAME", name="dense")
+
+        # The 1x1 conv to produce the skip contribution.
+        weights_skip = tf.Variable(tf.truncated_normal(
+            [1, dilation_channels, skip_channels], stddev=0.01),
+                                   name="skip")
+        skip_contribution = tf.nn.conv1d(out,weights_skip,stride=1,
+            padding="SAME", name="skip")
+
         if self.use_biases:
             biases_dense = tf.Variable(tf.constant(0.0,shape=[in_channels]),
```

Comment:
You could insert a space between `0.0,` and `shape` and a few lines below as well.